README.md

```diff
@@ -17,6 +65,15 @@ cosmoz-moment is a Polymer component for centralized management of Moment.js wit
 <link rel="import" href="bower_components/cosmoz-moment/cosmoz-moment.html" />
 ```
 
+### Use it inside your element
+```html
+<cosmoz-moment locale="[[locale]]"></cosmoz-moment>
+```
+
 ## Docs
```

Comment:
Docs will be handled by wc.org, not sure what their guidelines say about this, maybe omit it?